mock-server/controllers/group.ts

```diff
@@ -0,0 +1,70 @@
+import * as json from '../data/group.json'
+import find from '../utils/find'
+import error from '../utils/error'
+
+/**
+ * '/group/'
+ * @param {object} ctx
+ */
+const index = (ctx) => {
+  const files = ctx.query && ctx.query['files[]']
+  const publicKey = ctx.query && ctx.query.pub_key
+
+  if (!files || files.length === 0) {
+    return error(ctx, {
+      statusText: 'no files[N] parameters found.'
+    })
+  }
+
+  if (files && files.length > 0) {
+    for (let key in files) {
+      if (files.hasOwnProperty(key)) {
+        const file = files[key]
+        const array = file.split('/')
+        const uuid = array[0]
+
+        if (uuid.length < 36) {
```

Comment:
We can check uuid using regex

https://github.com/uploadcare/common/blob/ac2b10c3130cbdc0740e1b2ad2416ac7a772b017/uploadcare_common/regexps.py#L1

Reply:
Yes, thank you!